entrypoint/mixedoscluster/mixedoscluster_suite_test.go

```diff
@@ -0,0 +1,37 @@
+package mixedoscluster
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/factory"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var sonobuoyVersion string
+
+func TestMain(m *testing.M) {
+	flag.StringVar(&sonobuoyVersion, "sonobuoyVersion", "", "Sonobuoy Version that will be executed on the cluster")
+	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
+func TestMixedOSClusterCreateSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	RunSpecs(t, "Create Mixed OS Cluster Test Suite")
+}
+
+var _ = AfterSuite(func() {
```

Comment:
can you add on the after suite or somewhere else you think is suitable, a automatic cleanup of the sonobuoy? 
instead of only being called if i say delete caliing the mixedos_sonobuoy.sh ?

Reply:
I added that in the same script. Let me know if you think otherwise: `scripts/mixedos_sonobuoy.sh`